testsuite/objects/__init__.py

```diff
@@ -99,6 +99,33 @@ class Rule:
     value: str
 
 
+@dataclass
+class AnyPatterns:
```

Comment:
Plural does not seem right, but not sure what is a better name

Reply:
Renamed